src/app/server/CommissioningWindowManager.cpp

```diff
@@ -175,6 +175,12 @@ void CommissioningWindowManager::OnSessionEstablishmentStarted()
     // As per specifications, section 5.5: Commissioning Flows
     constexpr System::Clock::Timeout kPASESessionEstablishmentTimeout = System::Clock::Seconds16(60);
     DeviceLayer::SystemLayer().StartTimer(kPASESessionEstablishmentTimeout, HandleSessionEstablishmentTimeout, this);
+
+    ChipLogProgress(AppServer, "Commissioning session establishment step started");
+    if (mAppDelegate != nullptr)
+    {
+        mAppDelegate->OnCommissioningSessionEstablishmentStarted();
```

Comment:
should we be more specific and just call it `OnCommissioningPaseSessionEstablishmentStarted`?

Reply:
There are other pre-existing callbacks in the AppDelegate that do not mention the word PASE either. See: [https://github.com/project-chip/connectedhomeip/blob/master/src/app/server/AppDelegate.h](https://github.com/project-chip/connectedhomeip/blob/master/src/app/server/AppDelegate.h)

I am just following that pattern here, but can change all the callbacks if there is strong reason/desire for doing that.